### ISSUE TITLE
Question regarding lr in Neural Tangents Cookbook

### ISSUE THREAD

**Issue by alicedimo** [google/neural-tangents#184]
Hi,

I have a question regarding the "Training a neural network" section in the Neural Tangents Cookbook. In the code, lr=0.1 and if I change it to lr=0.01, the resulting graph shows the finite learning has shifted away from the linearised network on the x axis.

![image](https://github.com/google/neural-tangents/assets/37042544/3cb1e5de-59ef-4599-9dfe-01ba71825be6)

However, as far as I understood, the finite network is over-parameterized and decreasing the learning rate should improve the approximation of the linearized network.  So I was wondering why there is this shift?
And maybe a more general question; how do you account for the learning rate when going from gradient flow to gradient descent? (I hope the question is not too general, but I fear I misunderstood something in this process)

Thanks for any help!
